Makefile

```diff
@@ -5,7 +5,6 @@ include ./static/tools/mk/lint.mk
 
 GOBIN = build/bin
 GO ?= latest
-networkid ?= 
```

Comment:
Will the removal of `networkid` not affect it's usage down below?

Reply:
Nope, we dont need it commands like `make test-e2e networkid=4` work perfectly.